subcommand/server-acl-init/command_test.go

```diff
@@ -72,7 +72,8 @@ func TestRun_Defaults(t *testing.T) {
 }
 
 // Test the different flags that should create tokens and save them as
-// Kubernetes secrets.
+// Kubernetes secrets. We also test using with and without using the
+// -name-override to ensure the secrets are created with the right prefix.
 func TestRun_Tokens(t *testing.T) {
```

Comment:
I added another case in this test for running with/without the -name-override flag.